packages/flutter/pubspec.yaml

```diff
@@ -18,14 +18,14 @@ topics:
   - backend
 
 environment:
-  sdk: ">=2.19.6 <4.0.0"
-  flutter: ">=3.3.10"
+  sdk: ">=3.0.6 <4.0.0"
+  flutter: ">=3.10.6"
 
 dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^5.1.3
+  parse_server_sdk: ^6.4.0
```

Comment:
Yes ,this is a better method.

Reply:
I think you can upgrade now and then let's see if the CI passes.

Reply:
Due to the conflict that occurred in tests and [CI](https://github.com/parse-community/Parse-SDK-Flutter/actions/runs/8671176221/job/23780068144), I think it is better not to upgrade `parse_server_sdk` so that it is compatible with our version policy in the future.

Reply:
Due to the [conflicts ](https://github.com/parse-community/Parse-SDK-Flutter/actions/runs/8919290996/job/24495234767) in `flutter 3.10` with `parse_server_sdk 7.0.0`, it is not possible to upgrade parse_server_sdk until Flutter 3.10 is supported.

Reply:
I don't think it will cause any problem and you can merge this PR

Reply:
Did you upgrade the Parse Dart SDK? Why does it work now?